fastq_filtrator.py

```diff
@@ -0,0 +1,103 @@
+## Functions
+# This is a function that returns "Pass" if gc content of a read is between gc_bounds and "Fail" if it's not
+def gc_filter(seq, gc_bounds):
+    if len(gc_bounds) == 2:
+        min_gc, max_gc = int(gc_bounds[0]), int(gc_bounds[1])
+    elif len(gc_bounds) == 1:
+        min_gc, max_gc = int(gc_bounds[0]), 0 
+    else:
+        min_gc, max_gc = 0, 100
+    gc_count = int((seq.count('G') + seq.count('C')) * 100 / len(seq))
+    if ((gc_count < max_gc) and (gc_count > min_gc)):
+        return "Pass"
+    else:
+        return "Fail"
```

Comment:
1. Границы интервалов нужно было включать
2. Возвращать строки, которые по смыслу означают "Успех" и "Не успех" - плохая практика, лучше использовать логический тип
```suggestion
    return min_gc <= gc_count <= max_gc
```